Scripts/RemediationScripts/Remediate-EnableHTTPSForAppServices.ps1

```diff
@@ -37,30 +38,28 @@
 # Examples:
     To remediate:
         1. To review the App Services in a Subscription that will be remediated:
-    
            Enable-HttpsForAppServices -SubscriptionId 00000000-xxxx-0000-xxxx-000000000000 -PerformPreReqCheck -DryRun
 
         2. To enable HTTPS on the production slot and all non-production slots of all App Services in a Subscription:
-       
            Enable-HttpsForAppServices -SubscriptionId 00000000-xxxx-0000-xxxx-000000000000 -PerformPreReqCheck
 
         3. To enable HTTPS on the production slot and all non-production slots of all App Services in a Subscription, from a previously taken snapshot:
-       
            Enable-HttpsForAppServices -SubscriptionId 00000000-xxxx-0000-xxxx-000000000000 -PerformPreReqCheck -FilePath C:\AzTS\Subscriptions\00000000-xxxx-0000-xxxx-000000000000\202109131040\EnableHTTPSForAppServices\AppServicesWithoutHTTPSEnabled.csv
 
+        4. To enable HTTPS only on the production slot and all non-production slots of all App Services in a Subscription without taking back up before actual remediation:
+           Enable-HttpsForAppServices -SubscriptionId 00000000-xxxx-0000-xxxx-000000000000 -SkipBackup
+
         To know more about the options supported by the remediation command, execute:
-        
         Get-Help Enable-HttpsForAppServices -Detailed
 
     To roll back:
         1. To disable HTTPS on the production slot and all non-production slots of all App Services in a Subscription, from a previously taken snapshot:
            Disable-HttpsForAppServices -SubscriptionId 00000000-xxxx-0000-xxxx-000000000000 -PerformPreReqCheck -FilePath C:\AzTS\Subscriptions\00000000-xxxx-0000-xxxx-000000000000\202109131040\EnableHTTPSForAppServices\RemediatedAppServices.csv
         
-        2. To disable HTTPS on the production slot and all non-production slots of all App Services in a Subscription, from a previously taken snapshot:
+        2. To disable HTTPS on the production slot of all App Services in a Subscription, from a previously taken snapshot:
```

Comment:
why we removed non-production slot comments

Reply:
In the below command, we are passing a switch -ExcludeNonProductionSlots to exclude non-production slots.